src/uwtools/tests/drivers/test_ungrib.py

```diff
@@ -0,0 +1,184 @@
+# pylint: disable=missing-function-docstring,protected-access,redefined-outer-name
+"""
+Ungrib driver tests.
+"""
+import datetime as dt
+from unittest.mock import DEFAULT as D
+from unittest.mock import patch
+
+import yaml
+from pytest import fixture
+
+from uwtools.drivers import ungrib
+
+# Fixtures
+
+
+@fixture
+def cycle():
+    return dt.datetime(2024, 2, 1, 18)
+
+
+# Driver fixtures
+
+
+@fixture
+def config(tmp_path):
+    return {
+        "ungrib": {
+            "execution": {
+                "mpicmd": "srun",
+                "batchargs": {
+                    "nodes": 1,
```

Comment:
```suggestion
                    "cores": 1,
```
...if this works and `docs/shared/ungrib.yaml` likewise changes.